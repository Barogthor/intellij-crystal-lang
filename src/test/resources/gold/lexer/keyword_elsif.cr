if some_condition
  do_something
elsif some_other_condition
  do_something_else
else
  do_that
end