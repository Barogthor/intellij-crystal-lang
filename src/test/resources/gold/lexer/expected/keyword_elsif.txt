if ('if')
WHITE_SPACE (' ')
IDENT ('some_condition')
WHITE_SPACE ('\n  ')
IDENT ('do_something')
WHITE_SPACE ('\n')
elsif ('elsif')
WHITE_SPACE (' ')
IDENT ('some_other_condition')
WHITE_SPACE ('\n  ')
IDENT ('do_something_else')
WHITE_SPACE ('\n')
else ('else')
WHITE_SPACE ('\n  ')
IDENT ('do_that')
WHITE_SPACE ('\n')
end ('end')